### PR TITLE
DOCS: Initial new structure for presenting storage parameters, plus new table storage parameter

### PR DESCRIPTION
**Pull request by VMWareJennyBeth**
Staging here:

https://docs-staging.vmware.com/en/Greenplum-Database-7/improve-storage-param-docs/greenplum-database/GUID-ref_guide-sql_commands-CREATE_TABLE.html

https://docs-staging.vmware.com/en/Greenplum-Database-7/improve-storage-param-docs/greenplum-database/GUID-ref_guide-sql_commands-ALTER_TABLE.html

https://docs-staging.vmware.com/en/Greenplum-Database-7/improve-storage-param-docs/greenplum-database/GUID-ref_guide-sql_commands-CREATE_INDEX.html

### CODE REVIEW

File: gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md [hunk 1]
```diff
@@ -541,6 +542,29 @@ Move a table to a different schema:
 ALTER TABLE myschema.distributors SET SCHEMA yourschema;
 ```
 
+Change a table's access method to ao_row:
+
+```
+ALTER TABLE sales_1_prt_1 SET ACCESS METHOD ao_row;
```
Comment (soumyadeep2007) [inline gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md:548]: Better to use an example which doesn't have a partitioned table name like:
`ALTER TABLE distributors SET ACCESS METHOD ao_row;`

File: gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md [hunk 2]
```diff
@@ -541,6 +542,29 @@ Move a table to a different schema:
...
+```
+
+Change a table's blocksize to 32768:
+
+```
+ALTER TABLE SET (blocksize = 32768);
```
Comment (soumyadeep2007) [inline gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md:554]: Missing the tablename:
`ALTER TABLE distributors SET (blocksize = 32768);`

File: gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md [hunk 3]
```diff
@@ -541,6 +542,29 @@ Move a table to a different schema:
...
+```
+
+Change a table's access method to ao_row, compression type to zstd and compression level to 4:
+
+```
+ALTER TABLE sales_1_prt_1 SET ACCESS METHOD ao_row with (compresstype=zstd,compresslevel=4);
```
Comment (soumyadeep2007) [inline gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md:560]: sales_1_prt_1 -> sales

File: gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md [hunk 4]
```diff
@@ -541,6 +542,29 @@ Move a table to a different schema:
...
+```
+
+Change all future partitions of a table to have an access method of `heap`, leaving the access method of current partitions as is:
+
+ALTER TABLE ONLY sales_1_prt_1 SET ACCESS METHOD heap;
```
Comment (soumyadeep2007) [inline gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md:565]: sales_1_prt_1 -> sales

File: gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md [hunk 5]
```diff
@@ -541,6 +542,29 @@ Move a table to a different schema:
...
+```
+ALTER TABLE SET (blocksize = 32768);
+```
+
+Change a table's access method to ao_row, compression type to zstd and compression level to 4:
+
+```
+ALTER TABLE sales_1_prt_1 SET ACCESS METHOD ao_row with (compresstype=zstd,compresslevel=4);
+```
+
```
Comment (soumyadeep2007) [inline gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md:562]: We should add an example for changing all existing partitions of a table here:
ALTER TABLE sales SET ACCESS METHOD ao_row; -- considering sales is a partitioned table.

File: gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md [hunk 6]
```diff
@@ -541,6 +542,29 @@ Move a table to a different schema:
...
+Change all future partitions of a table to have an access method of `heap`, leaving the access method of current partitions as is:
+
+ALTER TABLE ONLY sales_1_prt_1 SET ACCESS METHOD heap;
+
```
Comment (soumyadeep2007) [inline gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md:566]: Let's add these 2 examples as well:

Adding a column and changing the access method at the same time:
ALTER TABLE distributors SET ACCESS METHOD ao_row, ADD column j int;

Adding a column and changing the storage parameters at the same time:
ALTER TABLE distributors SET (compresslevel=7), ADD COLUMN k int;

**Review by soumyadeep2007 — APPROVED** [review]
LGTM! Suggested a couple more examples.

File: gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md (hunk 1 above)
Comment (VMWareJennyBeth) [reply gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md:548]: Oof, did not clock that prt stood for partition! Thank you.

**Pull request closed without merging**